scrapers/getWorldometers.js

```diff
@@ -47,7 +47,7 @@ const mapRows = (_, row) => {
 				break;
 
 			default:
-				country[columns[index - 1]] = parseInt(cell.text().replace(/(\n|,)/g, '')) || 0;
+				country[columns[index - 1]] = cell.text() === '' ? null : (parseInt(cell.text().replace(/(\n|,)/g, '')) || 0);
```

Comment:
you could've basically just removed `|| 0` in the old line 50

Reply:
parseInt gives me NaNs when I do that

Reply:
Just noticed that although most missing values are empty strings, some are actually a space. Maybe this would be better?
```
isNaN(parseInt(cell.text().replace(/(\n|,)/g, ''))) ? null : parseInt(cell.text().replace(/(\n|,)/g, ''))
```